packages/fxa-auth-server/lib/routes/oauth/authorization.js

```diff
@@ -299,36 +302,84 @@ module.exports = ({ log, oauthDB, config }) => {
           strategy: 'sessionToken',
           payload: 'required',
         },
+        description: '🔒 sessionToken',
+        notes: [
+          "<p>🔒 HAWK-authenticated with session token</p><p>Authorize a new OAuth client connection to the user's account, returning a short-lived authentication code that the client can exchange for access tokens at the OAuth token endpoint.</p><p>This route behaves like the (oauth-server /authorization endpoint)[../fxa-oauth-server/docs/api.md#post-v1authorization] except that it is authenticated directly with a sessionToken rather than with a BrowserID assertion.</p>",
```

Comment:
Markdown link syntax is backwards.

```suggestion
          "<p>🔒 HAWK-authenticated with session token</p><p>Authorize a new OAuth client connection to the user's account, returning a short-lived authentication code that the client can exchange for access tokens at the OAuth token endpoint.</p><p>This route behaves like the [oauth-server /authorization endpoint](../fxa-oauth-server/docs/api.md#post-v1authorization) except that it is authenticated directly with a sessionToken rather than with a BrowserID assertion.</p>",
```